auto_submit/test/validations/revert_test.dart

```diff
@@ -79,6 +79,12 @@ void main() {
 
       Some other notes in the description a developer might add.
       And another note."""] = 'flutter/cocoon#123456';
+      tests['Pull request Reverts flutter/flutter#12334 is happening continuously.'] = 'flutter/flutter#12334';
```

Comment:
Here is the test where we still get the link but allow context before and after the match.